pyccel/stdlib/ndarrays/ndarrays.c

```diff
@@ -239,45 +239,47 @@ t_slice new_slice(int32_t start, int32_t end, int32_t step)
     return (slice);
 }
 
-t_ndarray array_slicing(t_ndarray arr, int n, ...)
+t_ndarray array_slicing(t_ndarray arr, int n, int n_slices, ...)
```

Comment:
Don't you always have `n_slices == arr.nd`?

Reply:
@EmilyBourne Can we close this conversation?